sda-commons-server-spring-data-mongo/src/main/java/org/sdase/commons/server/spring/data/mongo/SpringDataMongoBundle.java

```diff
@@ -219,6 +258,12 @@ private Validator createValidator() {
 
   /** shutdowns the spring application context, if created */
```

Comment:
Copy, paste error :D comment not updated!

Reply:
you guys don't let anything pass, I liked it 😮